open-machine-learning-jupyter-book/assignments/data-science/amazon-sagemaker-mlops-workshop-warm-up.md

```diff
@@ -0,0 +1,77 @@
+# Data Science
+
+Amazon SageMaker is a cloud service for Machine Learning provided by Amazon Web Services (AWS). It offers a complete platform that helps users build, train, and deploy Machine Learning models, along with a range of tools and algorithms that make Machine Learning more accessible and scalable.
+
+We will learn [amazon-sagemaker-mlops-workshop](https://github.com/awslabs/amazon-sagemaker-mlops-workshop/) provided by AWS Labs to help users understand the Machine Learning workflow on the Amazon SageMaker platform. The project covers a series of work steps, from data preparation to model development and deployment, and also presents best practices for managing Machine Learning workflows in production. The project is mainly divided into the following parts:
+
+1. Data Preparation: In this section, you will learn how to prepare a Machine Learning dataset and use Amazon SageMaker Data Labeling Tool for data labeling.
+2. Model Development: In this section, you will develop and train Machine Learning models using Amazon SageMaker Notebook instances, as well as optimize models using Amazon SageMaker Automatic Model Tuning feature.
+3. Model Evaluation and Interpretation: In this section, you will learn how to evaluate and interpret Machine Learning models using Amazon SageMaker model monitoring and interpretation features.
+4. Model Deployment and Management: In this section, you will learn how to deploy trained models to production environments using Amazon SageMaker model deployment feature, and manage Machine Learning workflows using Amazon SageMaker management feature.
+
+### Step 1 - initialize the cloud environment with CloudFormation 
+
+First, click to enter the open source project [amazon-sagemaker-mlops-workshop](https://github.com/awslabs/amazon-sagemaker-mlops-workshop/), and you can find the Launch Stack button in the Instructions section of the README.md file.
+
+![SageMaker guide 1](../../../images/SageMaker-guide-1.png)
+
+Click the Launch Stack button, which will create the required S3 dataset and SageMaker in the cloud, then create a subnet and set the corresponding permissions so that they can access each other.
+
+Then, enter [Amazon S3 URL](https://s3.amazonaws.com/aws-ai-ml-aod-latam/mlops-workshop/m.yml) to download the YAML file to the local.
+
+![SageMaker guide 2](../../../images/SageMaker-guide-2.png)
+
+After that, open the YAML file, since we do not need to use the high-end instance type like ml.m4.xlarge, we need to change InstanceType: "ml.m4.xlarge" to “ml.t2.medium” in the file.
+
+![SageMaker guide 3](../../../images/SageMaker-guide-3.png)
+
+```{note}
+If using ml.m4.xlarge, the notebook creation will not be successful.
+```
+
+Then, we can add the modified YAML file by clicking on the "Upload a template file" button.
+
+Subsequently, we need to set the NotebookInstanceSecGroupId parameter.
+
+![SageMaker guide 4](../../../images/SageMaker-guide-4.png)
```

Comment:
change the all file names to have only lowercase.

Reply:
ok